satpy/tests/utils.py

```diff
@@ -17,9 +17,27 @@
 """Utilities for various satpy tests."""
 
 from datetime import datetime
-from satpy.readers.yaml_reader import FileYAMLReader
 from unittest import mock
 
+from satpy.dataset import (DataID, DataQuery, default_id_keys_config,
+                           minimal_default_keys_config)
+from satpy.readers.yaml_reader import FileYAMLReader
+
+
+def make_dsid(**items):
+    """Make a data id."""
+    return DataID(default_id_keys_config, **items)
+
+
+def make_cid(**items):
+    """Make a composite ID."""
```

Comment:
What is a composite ID?  This seems to be also returning a Data ID.  Is a composite ID a Data ID with a particular minimal keys config?